src/engine/StartupActions.cs

```diff
@@ -25,7 +26,15 @@ private StartupActions()
             " latest log is 'log.txt'");
 
         // Load settings here, to make sure locales etc. are applied to the main loaded and autoloaded scenes
-        if (Settings.Instance == null)
-            GD.PrintErr("Failed to initialize settings.");
+        try
+        {
+            // We just want to do something to ensure settings instance is fine
+            // ReSharper disable once UnusedVariable
+            var hashCode = Settings.Instance.GetHashCode();
```

Comment:
Isn't it possible to just call the method and not save the return value in any variable to avoid that UnusedVariable warning?

Reply:
No, that gives a different warning. (return value of pure function is not used)